_targets.R

```diff
@@ -21,14 +29,22 @@ list(
 
   # PULL SITE DATA HERE
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png",
+                                        state_abb)),
     tar_target(nwis_inventory,
-               filter(oldest_active_sites, state_cd == state_abb) ),
-    tar_target(nwis_data, get_site_data(site_info = nwis_inventory,
-                                        state = state_abb,
-                                        parameter = parameter))
-    # Insert step for tallying data here
-    # Insert step for plotting data here
+      filter(oldest_active_sites, state_cd == state_abb)),
+    tar_target(nwis_data,
+      get_site_data(site_info = nwis_inventory,
+                    state = state_abb,
+                    parameter = parameter)),
+    tar_target(site_tallies,
+      tally_site_obs(nwis_data)),
+    tar_target(plot_sites,
```

Comment:
Small change -- this needs to be a file target, since `plot_site_data()` [generates a file and returns a file path](https://github.com/jesse-ross/ds-pipelines-targets-3/blob/appliers/3_visualize/src/plot_site_data.R#L12)

Reply:
Oh, good catch! Fixed now.